pkg/serverless/trace/inferredspan/inferred_span.go

```diff
@@ -72,3 +83,66 @@ func FilterFunctionTags(input map[string]string) map[string]string {
 
 	return output
 }
+
+func CreateInferredSpan(event string, inferredSpan InferredSpan) {
+	// Parse the event into the EventKey struct
+	eventSource, attributes := ParseEventSource(event)
+	switch eventSource {
+	case "apigateway":
+		CreateInferredSpanFromAPIGatewayEvent(eventSource, attributes, inferredSpan)
```

Comment:
The names of these functions confused me as well -- they all start with `create` but they don't seem to create anything